client/my-sites/media-library/google-photos-picker-button.tsx

```diff
@@ -0,0 +1,58 @@
+import { Button } from '@wordpress/components';
+import './google-photos-picker-button.scss';
+import { useTranslate } from 'i18n-calypso';
+import { useEffect } from 'react';
+import { useCreateGooglePhotosPickerSessionMutation } from 'calypso/data/media/use-google-photos-picker-session-mutation';
+import useGooglePhotosPickerSessionQuery from 'calypso/data/media/use-google-photos-picker-session-query';
+import { useDispatch, useSelector } from 'calypso/state';
+import { setPhotoPickerSession } from 'calypso/state/media/actions';
+import getGooglePhotosPickerSession from 'calypso/state/selectors/get-google-photos-picker-session';
+
+const GooglePhotosPickerButton = () => {
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+	const session = useSelector( getGooglePhotosPickerSession );
+	const { mutate: createSession, isPending } = useCreateGooglePhotosPickerSessionMutation();
+	const { data: sessionData, refetch } = useGooglePhotosPickerSessionQuery( session?.id );
+
+	const openPickerTab = () => {
+		session?.pickerUri && window.open( session.pickerUri, '_blank' );
+	};
+
+	useEffect( () => {
+		! session && createSession();
+	}, [ session, createSession ] );
+
+	useEffect( () => {
+		const interval = setInterval( refetch, 5000 );
+		return () => clearInterval( interval );
+	}, [ refetch ] );
+
+	useEffect( () => {
+		sessionData && dispatch( setPhotoPickerSession( sessionData ) );
+	}, [ sessionData ] );
+
+	return (
+		<div className="google-photos-picker--container media-library__connect-message">
+			<p>
+				{ translate(
+					'Select photos directly from your Google Photos library. Click the button below to open Google’s secure photo picker in a new tab.'
+				) }
+			</p>
+			<Button variant="primary" isBusy={ isPending } onClick={ openPickerTab }>
+				{ translate( 'Open Google Photos Picker' ) }
+			</Button>
+			<p>
+				<small>
+					(
+					{ translate(
+						"this page will automatically update when you've completed your selection"
```

Comment:
Maybe we can use a capital `T` in `this` here?

Reply:
The sentence is wrapped in brackets. If we add a first capital letter, we should add a dot at the end.

Should we ping someone from design?

<img width="639" alt="Screenshot 2024-11-04 at 11 38 18" src="https://github.com/user-attachments/assets/9be76959-01ae-4af2-8184-866c719628cb">

Reply:
@fditrapani could you help us review this once you have a second please? Would love your thoughts here. 🙏

Reply:
Thanks for the ping @ouikhuan. To answer your question directly, yes, we should capitalize T and add a period at the end as @ouikhuan suggested. I'd even remove the parenthesis since it's smaller copy:

> This page will automatically update when you complete your selection.

That being said, I think there's a lot of redundant text here. `Click on the button below to open Google's secure photo picker in a new tab` doesn't look necessary with the button copy reading `Open Google Photo Picker`. They say the same thing. We could add an icon to the button to communicate that it'll open a new tab:

<img width="354" alt="image" src="https://github.com/user-attachments/assets/1a149740-282a-4f7e-8dfc-274a79937e46">

The copy in parenthesis also doesn't seem helpful or necessary. I would assume that that would happen after I pick photos.

Reply:
Hey @fditrapani,

Thanks for the suggestion. I made changes and here you can see how it looks now.
<img width="1269" alt="Screenshot 2024-11-04 at 16 07 01" src="https://github.com/user-attachments/assets/879ab750-f62f-41b2-8812-a7ebf12c137e">

From my point of view, the screen looks quite empty. I tried to reuse the Google logo from the connection screen and add max paragraph size.
<img width="1268" alt="Screenshot 2024-11-04 at 16 06 04" src="https://github.com/user-attachments/assets/325343cf-ee9e-4308-b4a1-61463386af2d">

Let me know what do you think?

Reply:
Thanks for the screenshots @bogiii! I agree that it feels a bit empty and like your suggestion with the logo. Can we adjust the size of the logo and the spacing to match our other empty state screens:

![image](https://github.com/user-attachments/assets/85991261-79be-4afa-a94d-92bc59016723)


Here's another alternative that fits more inline with our other empty states:
![image](https://github.com/user-attachments/assets/ae874dbe-1709-4edd-a004-f1a7e878970d)

Reply:
Sounds good! I can adjust it following your screenshot suggestion.

Just to double-check. Are we going to update the copy as well or keep the previous suggestion?
1. "Select photos directly from your Google Photo library" or
2. "Click on the button below to open Google's secure photo picker in a new tab"

Reply:
My vote is to go with "Select photos directly from your Google Photo library". "Click on the button below to open Google's secure photo picker in a new tab" feels redundant with the button copy.

Reply:
Thanks! I've updated the screen to follow the existing empty state.

<img width="1364" alt="Screenshot 2024-11-04 at 23 43 09" src="https://github.com/user-attachments/assets/e921dcc4-299a-4396-bcee-94460f309be5">

Reply:
Thanks for the review and the suggestion @fditrapani !

Reply:
No problem. Love it!